corehq/toggles/__init__.py

```diff
@@ -1120,6 +1120,14 @@ def _ensure_valid_randomness(randomness):
     namespaces=[NAMESPACE_DOMAIN],
 )
 
+MULTI_VIEW_API_KEYS = StaticToggle(
```

Comment:
somewhat nit, but generally it's preferred to PR the new toggles separately, as this file is often touched frequently and is a source of conflicts when other folks are introducing new toggles. Is it possible for you to cherry-pick this commit into a separate PR while we take time to address the feedback on this PR? Thanks!

Reply:
Created a separate PR: https://github.com/dimagi/commcare-hq/pull/34352

Reply:
thanks!